Project.toml

```diff
@@ -63,7 +63,7 @@ Adapt = "2.0.2, 3.2"
 ArgParse = "1.1"
 ArtifactWrappers = "0.1.1"
 BenchmarkTools = "0.5"
-CLIMAParameters = "0.1"
+CLIMAParameters = "0.2"
```

Comment:
Shouldnt we also have CloudMicrophysics = "0.2" in the Project.toml?

Reply:
This _is_ the Project.toml. Do you mean the docs? If so, then yes 😂.

Reply:
I thought that ClimateMachine.jl now depends on CloudMicrophysics.jl. So I thought it should actually be in the dependencies here?

Reply:
i.e. we need CloudMicrophysics.jl to specify some of the multiphysics tendencies?

Reply:
https://github.com/CliMA/ClimateMachine.jl/blob/ck/up_climaparams/Project.toml#L13